### PR TITLE
They're on a circle.

### PR DESCRIPTION
**Pull request by jdangerx**
still shaky tho

### CODE REVIEW

File: hairmit.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ var Engine = Matter.Engine,
     Vector = Matter.Vector;
 
 // create engine
-var engine = Engine.create(),
+var engine = Engine.create({ constraintIterations: 1 }),
```
Comment (porterjamesj) [inline hairmit.js:16]: niceo

File: hairmit.js [hunk 2]
```diff
@@ -122,31 +130,36 @@ function project(point, angle, length){
...
-    pointA: pin,
-    stiffness: 1,
-    length: 0,
-    render: {
-      visible: false
-    }
-  }));
+  var pin = project(pointA, pinAngle, pinDist);
+
+  // Composite.add(composite, Constraint.create({
```
Comment (porterjamesj) [inline hairmit.js]: Delete this garbage

File: hairmit.js [hunk 3]
```diff
@@ -235,7 +239,7 @@ hairs.composites.forEach(makeCuttable);
 
 function cutAbove(parent, body) {
   var toDelete = parent.constraints.filter((c) => c.bodyB === body);
-  World.remove(parent, toDelete);
+  // World.remove(parent, toDelete);
```
Comment (porterjamesj) [inline hairmit.js]: Fix this

Comment (porterjamesj) [reply hairmit.js]: We need to delete things

Comment (porterjamesj) [reply hairmit.js]: ya dingus

**Review by porterjamesj — DISMISSED** [review]
Change this stuff

**Comment by jdangerx** [comment]
Oops I forgot to link the JIRA issue

**Pull request merged**